<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TenantsDialog</class>
 <widget class="QDialog" name="TenantsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>619</width>
    <height>420</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tenants</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QTreeWidget" name="trvTenants">
     <property name="maximumSize">
      <size>
       <width>208</width>
       <height>16777215</height>
      </size>
     </property>
     <column>
      <property name="text">
       <string>Tenants</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="3" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Tel:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="txtTel"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Name:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Is Active:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="cboIsActive">
       <item>
        <property name="text">
         <string>No</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Yes</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="6" column="0" colspan="2">
      <widget class="QTableView" name="tblLeases"/>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="txtName"/>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="txtEmail"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Sex:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Email:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="cboSex">
       <item>
        <property name="text">
         <string>Male</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Female</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="5" column="0" colspan="2">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Leases:</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="cmdAddNew">
       <property name="text">
        <string>Add New</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cmdSave">
       <property name="text">
        <string>Save</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cmdDelete">
       <property name="text">
        <string>Delete</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="cmdExit">
       <property name="text">
        <string>Exit</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>trvTenants</tabstop>
  <tabstop>txtName</tabstop>
  <tabstop>cboSex</tabstop>
  <tabstop>cboIsActive</tabstop>
  <tabstop>txtTel</tabstop>
  <tabstop>txtEmail</tabstop>
  <tabstop>cmdAddNew</tabstop>
  <tabstop>cmdSave</tabstop>
  <tabstop>cmdDelete</tabstop>
  <tabstop>tblLeases</tabstop>
  <tabstop>cmdExit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
